gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/tasks/CreateManifestTaskIntegrationSpec.groovy

```diff
@@ -601,8 +601,9 @@ class CreateManifestTaskIntegrationSpec extends GradleIntegrationSpec {
 
         then:
         result.task(":foo-server:createManifest").outcome == TaskOutcome.SUCCESS
+        result.task(":foo-api:configureProductDependencies").outcome == TaskOutcome.SUCCESS
         result.task(':foo-api:jar') == null
-        result.tasks.size() == 1
+        result.tasks.size() == 2
```

Comment:
Given that we're pushing for all internal projects to use gradle-consistent-versions, what do you think about having one integration tests that actually uses GCV?

Reply:
wrote an integration test ensure that recommended-product-dependencies can get a version out of consistent-versions' getVersion